automation/foxpass-ansible-setup/roles/foxpass-setup/handlers/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+# handlers file for foxpass-setup
+- name: daemon reload
+  systemd:
+    daemon_reload: yes
+
+- name: stop ssh service
+  systemd:
+    name: ssh
+    state: stopped
+
+- name: restart ssh
+  systemd:
+    name: ssh
+    state: restarted
+
+- name: restart nslcd
```

Comment:
nslcd/nscd are only for Debian based distros, RHEL based distros use sssd.  Make main.yml distro agnostic please